Makefile

```diff
@@ -1,6 +1,9 @@
 GOARCH := $(shell go env GOARCH)
 PWD=$(shell pwd)
 
+fmt: **/*.go
```

Comment:
Did you confirm that this works? I've never seen this syntax in makefiles before.